### PR TITLE
OCM-2342: Updated examples

### PR DESCRIPTION
**Pull request by sagidayan**
- In IDPs, removes tfvars file, and added variables.tf files
 - Added an aws region statement for the oidc config flows

### CODE REVIEW

**Review by bardielle — APPROVED** [review]

**Comment by nirarg** [comment]
/hold
Please change `cluster = "my-cluster"` in all relevant places
Add variable: cluster_id

**Comment by sagidayan** [comment]
/unhold

@nirarg Please re-review

File: examples/create_rosa_cluster/create_rosa_sts_cluster/oidc_configuration/oidc_provider/main.tf [hunk 1]
```diff
@@ -31,6 +31,10 @@ provider "ocm" {
   url   = var.url
 }
 
+provider "aws" {
+  region = var.cloud_region
```
Comment (nirarg) [inline examples/create_rosa_cluster/create_rosa_sts_cluster/oidc_configuration/oidc_provider/main.tf:35]: This change is related to another PR I guess
Please remove it

Comment (sagidayan) [reply examples/create_rosa_cluster/create_rosa_sts_cluster/oidc_configuration/oidc_provider/main.tf]: removed

**Pull request merged**